### ISSUE TITLE
Storyly Items reload problem

### ISSUE THREAD

**Issue by erdemildiz** [Netvent/storyly-mobile#58]
Storyly Items reload problem

## Environment Details
**Storyly SDK Version:** ---  1.9.2
**Device Model:** --- iPhone6 
**Device OS Version:** --- 12.4
On Modanisa App

## Expected Behavior
```
*** Assertion failure in -[Storyly.StorylyListCollectionView _endItemAnimationsWithInvalidationContext:tentativelyForReordering:animator:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKitCore_Sim/UIKit-3698.140/UICollectionView.m:5953
2021-02-24 17:47:37.981808+0300 Modanisa[66899:8674086] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'attempt to delete item 4 from section 0 which only contains 1 items before the update'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010eab18db __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x000000010e042ac5 objc_exception_throw + 48
	2   CoreFoundation                      0x000000010eab1662 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x000000010a06c76b -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 194
	4   UIKitCore                           0x000000011324face -[UICollectionView _endItemAnimationsWithInvalidationContext:tentativelyForReordering:animator:] + 4572
	5   UIKitCore                           0x000000011325af24 -[UICollectionView _endUpdatesWithInvalidationContext:tentativelyForReordering:animator:] + 71
	6   UIKitCore                           0x000000011325b267 -[UICollectionView _performBatchUpdates:completion:invalidationContext:tentativelyForReordering:animator:] + 434
	7   UIKitCore                           0x000000011325b092 -[UICollectionView _performBatchUpdates:completion:invalidationContext:tentativelyForReordering:] + 90
	8   UIKitCore                           0x000000011325b015 -[UICollectionView _performBatchUpdates:completion:invalidationContext:] + 74
	9   UIKitCore                           0x000000011325af6a -[UICollectionView performBatchUpdates:completion:] + 53
	10  Storyly                             0x000000010cf1a0ae block_destroy_helper + 35646
	11  Storyly                             0x000000010cf19bb6 block_destroy_helper + 34374
	12  Storyly                             0x000000010ceceb39 block_destroy_helper + 11353
	13  Storyly                             0x000000010cf44460 globalinit_33_2B252F656DA8F447E83B5C66DE369249_func46 + 3120
	14  Storyly                             0x000000010cf476d1 globalinit_33_2B252F656DA8F447E83B5C66DE369249_func46 + 16033
	15  Storyly                             0x000000010cecc04e block_destroy_helper + 366
	16  libdispatch.dylib                   0x0000000110d30d7f _dispatch_call_block_and_release + 12
	17  libdispatch.dylib                   0x0000000110d31db5 _dispatch_client_callout + 8
	18  libdispatch.dylib                   0x0000000110d3f080 _dispatch_main_queue_callback_4CF + 1540
	19  CoreFoundation                      0x000000010ea18a79 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	20  CoreFoundation                      0x000000010ea13126 __CFRunLoopRun + 2310
	21  CoreFoundation                      0x000000010ea124d2 CFRunLoopRunSpecific + 626
	22  GraphicsServices                    0x00000001169982fe GSEventRunModal + 65
	23  UIKitCore                           0x00000001139c0fc2 UIApplicationMain + 140
	24  Modanisa                            0x0000000104cf51db main + 75
	25  libdyld.dylib                       0x0000000110da6541 start + 1
	26  ???                                 0x0000000000000002 0x0 + 2
)
libc++abi.dylib: terminating with uncaught exception of type NSException
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'attempt to delete item 4 from section 0 which only contains 1 items before the update'
terminating with uncaught exception of type NSException
CoreSimulator 732.18.6 - Device: iPhone 6 (A29C353F-CA97-456C-B8C2-8677AF776E43) - Runtime: iOS 12.4 (16G73) - DeviceType: iPhone 6
```  

## Current Behavior
App was crash

**Comment by levent-oral** [comment]
Hello @erdemildiz,

We've tested the issue with initialization and `refresh` method. Can you give information about 'step-to-reproduce'?

Thanks,

**Comment by erdemildiz** [comment]
@levent-oral We are getting this issue on our development stage. Issue occurs when change country. Example: If You are in Turkey lang TR and you have 10 item in story. After you change the country as Saudi Arabia, AR(arabic) you just have 5 item in story. You send a request after change country we are getting above issue.

**Comment by levent-oral** [comment]
We've released (iOS SDK 1.9.3)[https://github.com/Netvent/storyly-mobile/blob/master/ios/RELEASENOTES.md#193] version to improve usage and prevent crash for runtime segmentation change cases.

**Comment by erdemildiz** [comment]
Thanks  @levent-oral
